tests/testthat/test_await.R

```diff
@@ -127,20 +127,20 @@ test_that("verbose produces correct messages", {
 
 test_that("call_once returns list with element called", {
   f <- function(id) return(list(state = "succeeded"))
-  expect_true(call_once(f, .id = 1, fname = "f")$called)
+  expect_true(call_once(f, 1, fname = "f")$called)
 
   f <- function(id) return(list(state = "partying instead"))
-  expect_false(call_once(f, .id = 1, fname = "f")$called)
+  expect_false(call_once(f, 1, fname = "f")$called)
 })
 
 test_that("if called, get_status(r$response) returns status", {
   f <- function(id) return(list(state = "succeeded"))
-  expect_equal(get_status(call_once(f, .id = 1, fname = "f")$response), "succeeded")
+  expect_equal(get_status(call_once(f, 1, fname = "f")$response), "succeeded")
 })
 
 test_that("call_once captures completed_states and status_keys", {
   f <- function(id) return(list(party_status = "at the party"))
-  expect_true(call_once(f, .id = 1, .success_states = "at the party",
```

Comment:
why do these changes need to be made?

Reply:
I had written `call_once` with an `.id` argument previously to facilitate vectorization previously in `await_all`. It's not needed. The change in the test removes the `.id` named parameter in `call_once`.